Created the User's UI

ElinorW opened the pull request:
Created the User's views. Content is static and no database connection is needed.

ElinorW committed:
Creation of User's UI

Creation of the user's homepage view

ElinorW committed:
Updated the user's homepage

ElinorW committed:
Updates to the user's homepage

modified the 'currently borrowed' section

ElinorW committed:
Implementing the route attribute

This is to separate the user's and admin's views

ElinorW committed:
Creation of the Books view on the User's UI

ElinorW committed:
Adding functionality to the buttons

ElinorW committed:
Updated admin routing

ElinorW committed:
Creation of the Office item's view on the User's UI

ElinorW committed:
Updates to the User's homepage

ElinorW committed:
Improving elements in the views

ElinorW committed:
Added a form input for uploading images

ElinorW committed:
Added some javascript

irvinesunday commented on AssetTracking/AssetTracking/Controllers/OfficeItemsController.cs:
```diff
@@ -6,13 +6,18 @@
 namespace AssetTracking.Controllers
 {
-    [Route("[controller]")]
+    //[Route("[controller]")]
```
Let's remove this unused code

ElinorW replied:
Alright

irvinesunday commented on AssetTracking/AssetTracking/Controllers/OfficeItemsController.cs:
```diff
@@ -6,13 +6,18 @@
         {
             return View("~/Views/Admin/OfficeItems.cshtml");
         }
+        [Route("[controller]")]
```
[nitpick] Fix formatting to add white spaces to separate the methods in your controller.

irvinesunday commented on AssetTracking/AssetTracking/Views/Shared/_Layout.cshtml:
```diff
@@ -48,7 +48,7 @@
     </header>
     <div class="container">
-        <partial name="_CookieConsentPartial" />
+        <!--<partial name="_CookieConsentPartial" />-->
```
Again... remove unused code

ElinorW committed:
Improving formatting and aesthetics

cwahome commented on AssetTracking/AssetTracking/Views/User/MyBooks.cshtml:
```diff
@@ -0,0 +1,102 @@
+        <p>Returned on : <span style="color:blue">30/06/2019 </span></p>
+    </div>
+</div>
+
```
Possibly consider using vertical tabs in place of the sections to prevent information overload on the user.

ElinorW replied:
Alright I will do that

ElinorW committed:
Added Tabs to the User's view 

User can view separate tabs of items he/she has currently borrowed and had previously borrowed

ElinorW committed:
Deleted commented code

ElinorW committed:
Merge branch 'dev' into ek/users-ui
